.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
>I want to preserve that bit that allows someone to pop a new one in the directory on the server and have it get served once todd-server is restarted.

That's where I'm confused. In [assets.go](https://github.com/Mierdin/todd/blob/9fdd982f36bd474d18501ff4bad7d8d03ad6a5e4/server/assets.go), it never reads and hashes files that already exist in the directories, so it would never instruct the agents to download them, right?